pkg/libcose/Makefile.include

```diff
@@ -10,3 +10,9 @@ endif
 
 # Declare pseudomodules here to be selfcontained
 PSEUDOMODULES += libcose_crypt_%
+
+ifneq (,$(filter cortex-m0%,$(CPU_ARCH)))
+  # There is a problem with the LLVM assembler, the M0(+) instruction set, and
+  # the assembly part of this package
+  TOOLCHAINS_BLACKLIST += llvm
+endif
```

Comment:
I do not have any error when building

    make -C tests/unittests tests-libcose BOARD=samr21-xpro distclean all TOOLCHAIN=llvm

With `arm-none-eabi-gcc 6.3.1` and `clag 3.8.0` on ubuntu xenial.

But it fails with qDSA though. So could have been a problem of parallel output in murdock maybe ?
It says llvm-as unrecognized option -mcpu=cortex-m0plus. Same on arch linux.

Reply:
Murdock is happy when I removed `qDSA` https://ci.riot-os.org/RIOT-OS/RIOT/9398/44552886b429f1ce7c704249beb43964fed7a297/output/compile/tests/unittests/samr21-xpro:llvm.txt so it should be `pkg/qDSA` disabled instead.

    make -C tests/unittests tests-qDSA BOARD=samr21-xpro distclean all TOOLCHAIN=llvm

I disabled the test in DISABLE_TEST_FOR_ARM_CORTEX_M for unittests.

I will re-enable the qDSA test to verify.

Reply:
RIOT/tests/unittests/bin/pkg/samr21-xpro/qDSA/arm/asm/bigint_mul.S:17:3: error: invalid instruction, any one of the following would fix this:
      add r0,#8